### PR TITLE
Changes to TPM 3.0.0 and above

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

**Comment by isrumnon** [comment]
@JohanFreelancer9 @get-itips @tseward @yogkumgit can you please review? Need to get this doc updated asap

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -51,7 +60,7 @@ Users homed on Skype for Business Online will a TenantId that is equal to some v
 
 **Note:**
 
-Beginning Microsoft Teams PowerShell version 2.6.2 onwards, the below updates are applicable for TeamsOnly customers.
+The below updates are applicable for TeamsOnly customers using Microsoft Teams PowerShell version 3.0.0 and above.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:63]:
```suggestion
The following updates are applicable for TeamsOnly customers using Microsoft Teams PowerShell version 3.0.0 and above.
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -175,17 +239,17 @@ Beginning Microsoft Teams PowerShell version 2.6.2 onwards, the below updates ar
...
-  - AssignedPlans eq 'MCOEV' - For exact match
-  - AssignedPlans eq '*MCO*' - for contains checks.
- 
+- Assigned Plan filter - Previous format will no longer be supported. Existing filters like `AssignedPlan -eq '<some-xml-string>'` will stop working. This will need to be modified to one of the following formats:
+  - AssignedPlans -eq '*MCO' or 'MCO*'or '*MCO*' - for contains checks.
+The output format has also changed, the xml object is now a json object.
+
 - EnterpriseVoiceEnabled filter
-  - EnterpriseVoiceEnabled eq true / false
+  - EnterpriseVoiceEnabled -eq true / false
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:252]:
```suggestion
  - EnterpriseVoiceEnabled -eq true/false
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the minor suggested edit. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by isrumnon** [comment]
Hi @yogkumgit can you help with getting this PR merged and published?

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
close > reopen

**Pull request merged**